pkg/util/util.go

```diff
@@ -258,3 +260,23 @@ func AdjustDuration(v *time.Duration, defValue time.Duration) {
 		*v = defValue
 	}
 }
+
+// CreateCfgFile creates and writes b as config file to dir prefix
+func CreateCfgFile(b []byte, dir, prefix string) (*os.File, error) {
```

Comment:
This is an example of how a file can be written for testing: https://github.com/pingcap/dm/pull/212/files#diff-fa28104e876d9e8e832e695718aa8edeR86-R98

See if you can rewrite the test so that you don't need to add this new function.

Reply:
Revised.